cli/QuickstepCli.cpp

```diff
@@ -150,6 +154,14 @@ DEFINE_bool(initialize_db, false, "If true, initialize a database.");
 DEFINE_bool(print_query, false,
             "Print each input query statement. This is useful when running a "
             "large number of queries in a batch.");
+DEFINE_string(profile_file_name, "",
```

Comment:
Actually, I’d argue that this is a *feature*. We can build with profiling included, but enable it only when we want to (rather than doing so for every run). 


> On Jun 13, 2016, at 11:38, Harshad Deshmukh <notifications@github.com> wrote:
> 
> In cli/QuickstepCli.cpp <https://github.com/apache/incubator-quickstep/pull/28#discussion_r66825014>:
> 
> > @@ -150,6 +154,14 @@ DEFINE_bool(initialize_db, false, "If true, initialize a database.");
> >  DEFINE_bool(print_query, false,
> >              "Print each input query statement. This is useful when running a "
> >              "large number of queries in a batch.");
> > +DEFINE_string(profile_file_name, "",
> One thing which we can do is generate a default profile name if none is specified. The default profile name can be something like "quickstep-profile-.prof", so that name collisions are avoided. That way, we can get rid of the check !quickstep::FLAGS_profile_file_name.empty() in Line 450. It's ok if we don't do it right away, in which case you can leave a TODO note.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub <https://github.com/apache/incubator-quickstep/pull/28/files/00955f19652fe54468377e932365d77a13cc69c8#r66825014>, or mute the thread <https://github.com/notifications/unsubscribe/ACZB6-MkaeSHdvw6rXb4W8-5AHNsAr4iks5qLYeEgaJpZM4Iz4mC>.
>